openstack/loadbalancer/v2/flavorprofiles/requests.go

```diff
@@ -0,0 +1,110 @@
+package flavorprofiles
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// LIST
+
+type ListOptsBuilder interface {
+	ToFlavorProfileListQuery() (string, error)
+}
+
+type ListOpts struct {
+	Fields []string `q:"fields"`
+}
+
+func (opts ListOpts) ToFlavorProfileListQuery() (string, error) {
+	q, err := gophercloud.BuildQueryString(opts)
+	return q.String(), err
+}
+
+func List(c *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pager {
+	url := rootURL(c)
+	if opts != nil {
+		query, err := opts.ToFlavorProfileListQuery()
+		if err != nil {
+			return pagination.Pager{Err: err}
+		}
+		url += query
+	}
+	return pagination.NewPager(c, url, func(r pagination.PageResult) pagination.Page {
+		return FlavorProfilePage{pagination.LinkedPageBase{PageResult: r}}
+	})
+}
+
+// CREATE
+
+type CreateOptsBuilder interface {
+	ToFlavorProfileCreateMap() (map[string]interface{}, error)
+}
+
+type CreateOpts struct {
```

Comment:
Same here, it would be nice to have documentation. This comment is also valid for the other structs in this PR.